lib/internal/streams/utils.js

```diff
@@ -179,7 +179,7 @@ function isReadableErrored(stream) {
 
 function isClosed(stream) {
   if (!isNodeStream(stream)) {
-    return null;
+    return stream?.state === 'closed' ? true : null;
```

Comment:
Add a new branch for web streams instead.

Reply:
updating to remove these because these functions wont be reached on webstreams anyway

Reply:
Done